plugins/webhooks/server/index.ts

```diff
@@ -5,7 +5,25 @@ import WebhookProcessor from "./processors/WebhookProcessor";
 import CleanupWebhookDeliveriesTask from "./tasks/CleanupWebhookDeliveriesTask";
 import DeliverWebhookTask from "./tasks/DeliverWebhookTask";
 
-PluginManager.register(PluginType.API, webhookSubscriptions, config)
-  .registerProcessor(WebhookProcessor)
-  .registerTask(DeliverWebhookTask)
-  .registerTask(CleanupWebhookDeliveriesTask);
+PluginManager.add([
+  {
+    ...config,
```

Comment:
Do we even need config spread on all these? 🤔

Reply:
We shouldn't, but since `id` is required in type `Plugin`, it's to please the `tsc`. Agree that it's confusing though. By the way, the aforementioned idea eliminates this problem without an off-putting hack 😉